settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Avona Voice Reference Design",
-    "project": "Avona",
+    "title": "XCORE Voice Framework",
```

Comment:
`XCORE` or `XMOS`?  I don't much care one way or the other, but right now both are in use and I do not think that the inconsistency is a good idea.

Reply:
XCORE is used in the title for IO and Core frameworks so decided to follow the same convention.